src/app/contextBroker/contextBroker.cpp

```diff
@@ -344,7 +344,6 @@ static const char* validLogLevels[] =
 *   RequestType   request     - The type of the request
```

Comment:
Apart for the entry mentioned in https://github.com/apinf/fiware-orion/pull/34#discussion_r206085806, should we add an entry in CNR regarding this refactor? In the past (see https://github.com/telefonicaid/fiware-orion/releases/tag/1.13.0) we added...

Reply:
It is a pretty important refectoring, but ... only internal stuff.
Are our users really interested in this kind of things?

Reply:
> Are our users really interested in this kind of things?

I have a similar doubt... and it case of doubt I tend to prefer to include ;) Specially when we did the same for 1.13.0 release.

Reply:
Sure, no probs. Npt 100% sure how to describe it though ... Wont talk about the "payloadWord", that's for sure! :-D

Reply:
Fixed in 1e2e62cef